content/docs/rosa/rosa-gitops/_index.md

```diff
@@ -0,0 +1,372 @@
+
+This step sets needed environment variables that are necessary to continue the walkthrough:
+
+* `AWS_ACCOUNT_ID`: the AWS account ID where the ROSA cluster will be provisioned.
+* `ROSA_CLUSTER_NAME`: the ROSA cluster name that the OCM operator will be installed upon.
+* `ROSA_USER_ROLE`: the [user role](https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-understanding-user-role_rosa-sts-about-iam-resources) 
+that is associated with the cluster.  This is a mandatory input to the provisioning process.
+* `OCM_OPERATOR_NAMESPACE`: the namespace where the ocm-operator will be installed.
+* `OCM_OPERATOR_VERSION`: the version of ocm-operator that will be installed.
+
+```bash
+export AWS_ACCOUNT_ID=111111111111
+export ROSA_CLUSTER_NAME=dscott
+export ROSA_USER_ROLE="arn:aws:iam::${AWS_ACCOUNT_ID}:role/ManagedOpenShift-User-dscott_mobb-Role"
+export OCM_OPERATOR_NAMESPACE=ocm-operator
+export OCM_OPERATOR_VERSION=v0.1.0
+```
+
+
+## Install the OCM Operator
+
+
+### Retrieve and Store the OCM Token
+
+1. Create a namespace where you wish to install the operator:
+
+```bash
+oc new-project $OCM_OPERATOR_NAMESPACE
+```
+
+2. Create a secret containing the OCM_TOKEN.  This token can be obtained form 
+https://console.redhat.com/openshift/token and is used by the operator to authenticate 
+against the OCM API.  This token must exist in the same namespace that the operator 
+is running and be named `ocm-token`.  It also expects the key to be called `OCM_TOKEN` 
+as the operator is expecting this value as an environment variable.  **Be sure to 
+substitute ${MY_OCM_TOKEN} with your token in the below command!**
+
+```bash
+oc create secret generic ocm-token \
+  --namespace=$OCM_OPERATOR_NAMESPACE \
+  --from-literal=OCM_TOKEN=${MY_OCM_TOKEN}
+```
+
+
+### Create the IAM Policies and Roles
+
+The operator will need to elevate privileges in order to perform things like 
+creating the operator-roles for the clusters.  Because of this, the operator 
+must have a specific role created to allow it these permissions.  In each instance, 
+it is a best practice to create a new set of policies and roles for each instance 
+of the OCM Operator.  Policies and roles are prefixed with the `ROSA_CLUSTER_NAME` 
+environment variable that is specified below.
+
+**NOTE:** please understand what you are doing if you deviate from the known good 
+policies.  If errors or more stringent security lockdowns are found, please submit a PR 
+so that we can get this fixed.
+
+1. Run the script to create the required policies and roles.  This creates a permission boundary, 
+a policy for the operator, and a role which allows the operator to assume a role against the OIDC 
+identity of the ROSA cluster.  If the policies and roles already exist (prefixed by your cluster 
+name), then the creation of them is skipped:
+
+```bash
+curl -s https://raw.githubusercontent.com/rh-mobb/ocm-operator/main/test/scripts/generate-iam.sh | bash
+```
```

Comment:
I'm not sure if I'm super-comfortable with this. Can we do this is a safer way?

Reply:
The manual steps to do this are far more intrusive and error prone than this.  I am putting together terraform for it now if you prefer to include that.

Reply:
That'd be great, but to unblock this PR, I think it'd be okay with just changing this so we download the file and chmod it. I mostly don't like just automatically passing through instructions to the shell from the internet.

Reply:
@Conan-Kudo Should be fixed now.  Let me know if this works for you:

https://github.com/rh-mobb/documentation/pull/377/commits/f12c0206f838d9de967d367d0c595a669ac9a150